packages/salesforcedx-vscode-lwc/src/messages/i18n.ja.ts

```diff
@@ -37,11 +37,11 @@ export const messages = {
   force_lightning_lwc_stop_in_progress: 'ローカルの開発サーバを停止しています',
   force_lightning_lwc_preview_text:
     'SFDX: コンポーネントをローカルでプレビュー',
-  force_lightning_lwc_preview_file_undefined:
```

Comment:
Renaming these since they're now shared across commands and are not exclusive to the Preview command anymore.